### PR TITLE
[OSDOCS#9716] GA OpenStack with rootVolume and etcd on local disk 

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.17

Issue: [OSDOCS-9716](https://issues.redhat.com//browse/OSDOCS-9716)

Link to docs preview: https://81714--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.html

QE review:
- [x] QE has approved this change.

Additional information: #81888 is merged. I think I'll just end up incorporating #81889 into this.

### CODE REVIEW

**Comment by maxwelldb** [comment]
Will incorporate https://github.com/openshift/openshift-docs/pull/81888 and https://github.com/openshift/openshift-docs/pull/81889

**Comment by rlobillo** [comment]
Hello @maxwelldb.

Please see below some comments:

- I think there is another warning that should be removed: https://github.com/openshift/openshift-docs/blob/main/modules/installation-osp-local-disk-deployment.adoc?plain=1#L11

- The 98-var-lib-etcd.yaml has a difference in comparison to the TP. We need to add "Requisite=var-lib-etcd.mount"
in the relabel-var-lib-etcd.service definition [here](https://github.com/openshift/openshift-docs/blob/main/modules/installation-osp-local-disk-deployment.adoc?plain=1#L155). Please follow [this](https://git.app.eng.bos.redhat.com/git/openshift-ir-plugin.git/tree/roles/day2ops/files/98-var-lib-etcd.yaml)  which si the one that we are testing on CI.

**Comment by maxwelldb** [comment]
I'll cherry-pick the current state of the related bugfix (77417e499e0785c03aca3a328b8d478bd12b6a0f) while I roll your suggestions into this. The preview should update shortly after that, assuming Prow stays functional. 🤞

File: modules/installation-osp-local-disk-deployment.adoc [hunk 1]
```diff
@@ -8,10 +8,7 @@
 
 If you have an existing {rh-openstack} cloud, you can move etcd from that cloud to a dedicated ephemeral local disk.
 
-[WARNING]
-====
-This procedure is for testing etcd on a local disk only and should not be used on production clusters. In certain cases, complete loss of the control plane can occur. For more information, see "Overview of backup and restore operation" under "Backup and restore".
-====
+// TODO: Verify that we no longer need any sort of warning here.
```
Comment (maxwelldb) [inline modules/installation-osp-local-disk-deployment.adoc:11]:
```suggestion
```

**Comment by maxwelldb** [comment]
@rlobillo Updated. Preview link in PR description should have the latest after CI passes.

**Comment by rlobillo** [comment]
@maxwelldb I see another warning on line 75, that's the one I wanted to mention before :sweat:

**Review by cbippley — COMMENTED** [review]
LGTM!

**Pull request merged**